rules/windows/defense_evasion_stop_process_service_threshold.toml

```diff
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2020/12/03"
+maturity = "production"
+updated_date = "2020/12/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule identifies high number of evasion related process executions (stop, delete or suspend a process or service)
+from the same host and within a short time period. This may indicate defense evasion attempt.
```

Comment:
```suggestion
This rule identifies a high number (10) of process executions (stop, delete, or suspend)
from the same host within a short time period. This may indicate a defense evasion attempt.
```

Reply:
re-lint